tests/utils/parsers/test_parser_instance_options.py

```diff
@@ -0,0 +1,32 @@
+import unittest2
+
+from tuned.utils.parsers.parser_instance_options import ParserInstanceOptions
+from tuned.plugins.base import Plugin
+
+class ParserInstanceOptionsTestCase(unittest2.TestCase):
```

Comment:
Add the test case in the same commit where you add the ParserInstanceOptions class.